### ISSUE TITLE
Missing information at installation using Jar file

### ISSUE THREAD

**Issue by kandovit** [realm/realm-java#1484]
Hi,

I've just added the Realm to my repositories using the second method (Jar) mentioned in your guide (https://realm.io/docs/java/latest/#installation) but I didn't manage to success due to missing information. 
I've added, as step 3, the Jar file and then, as step 4, Sync Project with Gradle Files but nothing happened. I had to enter Project Stracture --> App --> Dependencies and add there "File dependency", and only after I've sync the project I've managed to success.

**Comment by cmelchior** [comment]
What version of Android Studio are you using? 
I just tried creating a new project on `1.4 beta 4` where following the instructions on the website did work, meaning that I was able to reference Realm files and compile the project after pressing `Sync Project with Gradle Files`

**Comment by kneth** [comment]
@kandovit Do you still experience this problem after you have sync'ed your project with your Gradle files?

**Comment by kneth** [comment]
I'm closing the issue. @kandovit if the issue still persist, don't hesitate either create a new issue or reopen.
